jobs/alertmanager/spec

```diff
@@ -24,6 +24,42 @@ consumes:
     type: alertmanager
 
 properties:
+# For more informations: https://github.com/prometheus/alertmanager/blob/b5d1a64ad5bb0ff879705714d1e40cea82efbd5c/docs/configuration.md#jira_config
```

Comment:
From the document you linked it looks like only the `alertmanager.jira.url` needs to be configured within the global section of the [alertmanager.yml](https://github.com/prometheus/alertmanager/blob/b5d1a64ad5bb0ff879705714d1e40cea82efbd5c/docs/configuration.md#file-layout-and-global-settings). 

All other settings need to get configured when configuring a [receiver](https://github.com/prometheus/alertmanager/blob/b5d1a64ad5bb0ff879705714d1e40cea82efbd5c/docs/configuration.md#receiver). 

That means you can remove everything beside `alertmanager.jira.url`. Please move that spec right above the opsgenie spec so it follows the same structure as the config you sent.